hack/deploy-seed.sh

```diff
@@ -1,8 +1,24 @@
 #!/usr/bin/env bash
 
-echo ""
-echo "Installing Minio"
-helm --namespace mla upgrade --atomic --create-namespace --install minio charts/minio --values config/minio/values.yaml
+if [ "$1" = "-h" ]; then
+   cat << EOF
+Usage: deploy-seed.sh [-skip-minio]
+
+Deploy MLA backend components on a KKP Seed cluster.
+-skip-minio flag can be used if you want to configure MLA backend components to user existing minio in the cluster, if this flag is not specified, it will deploy a minio instance for you.
+EOF
+   exit 0
+fi
+
+if [ "$1" != "-skip-minio" ]; then
+    echo ""
+    echo "Installing Minio"
+    helm --namespace mla upgrade --atomic --create-namespace --install minio charts/minio --values config/minio/values.yaml
+
+    echo ""
```

Comment:
I am not sure about the lifecycle manager - it can be still useful if an external Minio is used (otherwise old data will not be automatically deleted).
But it makes sense to not deploy it if other (non-Minio) external s3 store is used.

Reply:
Ok, I see, then maybe I will just move this life cycle manager out of the if statement for now?

Reply:
yes, either that, or create separate `minio-lifecycle-mgr` arg

Reply:
I have added another flag `--skip-minio-lifecycle-mgr`, let me know if anything is missing